pages/common/license.md

```diff
@@ -2,18 +2,18 @@
 
 > Create license files for open-source projects.
 
-- Create a license:
+- Print a license to stout, using the defaults (auto-detected author name, and current year):
```

Comment:
Hahah nice catch! I must have been thinking of the dinner I'm planning for tonight :P